client.go

```diff
@@ -1208,7 +1207,7 @@ func (c *Client) PerformRequestC(ctx context.Context, method, path string, param
 		if ctx == nil {
```

Comment:
v5 [doesn't have that `ctx == nil`](https://github.com/olivere/elastic/blob/release-branch.v5/client.go#L1253). We need to review if we can use the same `PerformRequest` as v5 in v3.